src/PolyhedralGeometry/PolyhedralFan/properties.jl

```diff
@@ -212,8 +212,22 @@ julia> cones(PF, 2)
 """
 function cones(PF::_FanLikeType, cone_dim::Int)
   l = cone_dim - length(lineality_space(PF))
-  l < 1 && return nothing
-  return SubObjectIterator{Cone{_get_scalar_type(PF)}}(
+  t = Cone{_get_scalar_type(PF)}
+  l < 0 && return _empty_subobjectiterator(t, PF)
```

Comment:
```suggestion
  (l < 0 || dim(PF) == -1) && return _empty_subobjectiterator(t, PF)
```
The empty fan should also return an empty list.